consensus/broadcast/delayedBroadcast.go

```diff
@@ -35,16 +36,16 @@ type ArgsDelayedBlockBroadcaster struct {
 	AlarmScheduler        timersScheduler
 }
 
-type validatorHeaderBroadcastData struct {
-	headerHash           []byte
-	header               data.HeaderHandler
-	metaMiniBlocksData   map[uint32][]byte
-	metaTransactionsData map[string][][]byte
-	order                uint32
-	pkBytes              []byte
+type ValidatorHeaderBroadcastData struct {
+	HeaderHash           []byte
+	Header               data.HeaderHandler
+	MetaMiniBlocksData   map[uint32][]byte
+	MetaTransactionsData map[string][][]byte
+	Order                uint32
+	PkBytes              []byte
 }
 
-type delayedBroadcastData struct {
+type DelayedBroadcastData struct {
```

Comment:
is there a reason for the struct to be exported?

Reply:
to be, also, accessible in delayedBroadcasterMock